apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -280,3 +287,33 @@ export function EnrollLtiMessage({ ltiInfo, resLocals }: EnrollLtiMessageProps)
     </html>
   `.toString();
 }
+
+export function EnrollmentLimitExceededMessage({ resLocals }: { resLocals: Record<string, any> }) {
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", {
+          ...resLocals,
+          navPage: 'enroll',
+          pageTitle: 'Courses',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'enroll',
+        })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-danger text-white">Enrollment limit exceeded</div>
+            <div class="card-body">
+              This course has reached its enrollment limit. Please contact the course staff for more
+              information.
```

Comment:
Do we want to give their emails? Or at least UIDs until we have emails.

Reply:
IMO, no. Given that this is a student-initiated action, they should already know who is running their course? And a few more reasons:

- In practice, we see people adding "random" (trusted, but not actually on course staff) people as owners. We don't want them getting random emails.
- In a similar vein, this is different from the recent changes in PT that we're probably both thinking about, where identity is deliberately revealed by the person making invites.

Reply:
Yeah, I agree that in practice students will know where to complain 😄